lib/drivers/animasDriver.js

```diff
@@ -1157,12 +1157,16 @@ module.exports = function (config) {
           t.setSeconds(t.getSeconds() + 30);
           t = t.toISOString();
 
+          var dt = new Date(bolusdatum.deviceTime);
+          dt.setSeconds(dt.getSeconds() + 30);
```

Comment:
What's this addition of 30 secs for? We shouldn't have to do anything to `deviceTime` for de-duping...